src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -196,20 +199,39 @@ && isNotificationEventValidToSendToOtherPartySubscription(otherParty.getOtherPar
         return otherPartySubscription;
     }
 
+    private boolean canSendBasedOnConfidentiality(String partyMember) {
+        if (!(DIRECTION_ISSUED.equals(getNotificationType()) || DIRECTION_ISSUED_WELSH.equals(getNotificationType()))) {
+            return true;
+        }
+
+        String confidentialityType = responseWrapper.getNewSscsCaseData().getConfidentialityType();
+        List<String> confidentialityPartyMembers = responseWrapper.getNewSscsCaseData().getConfidentialityPartyMembers();
+
+        if (isNull(confidentialityType)
+                || isNull(confidentialityPartyMembers)
+                || "general".equalsIgnoreCase(confidentialityType)) {
+            return true;
+        }
+
+        return confidentialityPartyMembers.contains(partyMember);
+    }
 
     private boolean isNotificationEventValidToSendToAppointee() {
-        return hasAppointeeSubscriptionOrIsMandatoryAppointeeLetter(responseWrapper)
+        boolean isValid = hasAppointeeSubscriptionOrIsMandatoryAppointeeLetter(responseWrapper)
             && (EVENTS_VALID_FOR_ALL_ENTITIES.contains(getNotificationType())
             || EVENTS_VALID_FOR_APPOINTEE.contains(getNotificationType())
             || isValidProcessHearingRequestEventForParty(PartyItemList.APPELLANT)
             || isValidRequestInfoIncompleteEventForParty(PartyItemList.APPELLANT));
+        return isValid && canSendBasedOnConfidentiality("appellantOrAppointee");
     }
 
     private boolean isNotificationEventValidToSendToAppellant() {
-        return (getOldSscsCaseData() != null && isValidReviewConfidentialityRequest(getOldSscsCaseData().getConfidentialityRequestOutcomeAppellant(), getNewSscsCaseData().getConfidentialityRequestOutcomeAppellant()))
+        boolean isValid =  (getOldSscsCaseData() != null
```

Comment:
Similar to above with NotificationType, could we get this old case data once, set as an object, then reuse it?